assets/sum-my-subscriptions/happy_flows.md

```diff
@@ -0,0 +1,83 @@
+# Sequence diagrams of the happy flow
+
+## "Hello! What can you do?"
+
+I am a user who is asking about the bots capabilities
+
+### Find tasks from question
+```mermaid
+sequenceDiagram
+    Frontend -->> Director: "Hello! What can you do?"
+    Director -->> LLM: "Is this a processable question?"
+    LLM -->> Director: "No, return this generic response"
```

Comment:
should LLM here be `tasks.py` / something descriptive like `planner` ? Or are we just describing the flow to any LLM

Reply:
ignore me, I spoke too soon. I can see `taskAgent` below